README.rst

```diff
@@ -0,0 +1,139 @@
+FIXME - testing correctness of these links
+.. image:: https://travis-ci.org/diffpy/diffpy.pdfmorph.svg?branch=master
+   :target: https://travis-ci.org/diffpy/diffpy.pdfmorph
+
+.. image:: http://codecov.io/github/diffpy/diffpy.pdfmorph/coverage.svg?branch=master
+   :target: http://codecov.io/github/diffpy/diffpy.pdfmorph?branch=master
+
+
+PDFmorph
+========================================================================
+# FIXME - include uses, unique features
+PDFmorph is a Python software package designed to increase the insight
+researchers can get into spectra collected for atomic pair distribution 
+functions.
+
+REQUIREMENTS
+------------------------------------------------------------------------
+
+PDFmorph requires Python 3 and several third party libraries 
+that are used by PDFmorph and its components.
+
+* NumPy              - library for scientific computing with Python
+* matplotlib         - Python 2D plotting library
+* SciPy              - library for highly technical Python computing
+* diffpy.utils       - shared helper utilities for wx GUI, 
+https://github.com/diffpy/diffpy.utils
+
+# FIXME - add link for instructions
+If you can't use conda and need to build from the sources, instructions are [here].
+
+INSTALLATION
+------------------------------------------------------------------------
+
+PDFmorph is an application which makes use of the Command Line Interface to
+run. If you are unfamiliar with the CLI, consult online resources on how to get 
+started with it on your machine.
+
+It is recommended that you use Anaconda Python to run this application; to do so,
+first make sure that you have installed it from Anaconda distribution here:
+https://www.anaconda.com/distribution/.
+
+The preferred method is to use Anaconda Python and install from the
+"diffpy" channel of Anaconda packages ::
+
+     conda config --add channels diffpy
+     conda install diffpy.pdfmorph
+
+# FIXME
+To ensure the installation worked, activate the relevant conda environment and 
+type ::
+
+	pdfmorph --version
+
+If installed correctly, this command should return PDFmorph's current version
+number.
+
+With Anaconda, PDFmorph can be later upgraded to the latest released
+version using ::
+
+     conda update diffpy.pdfmorph
+
+#FIXME - easy_install?
+With other Python distributions the program can be upgraded to
+the latest version as follows ::
+
+     easy_install --upgrade diffpy.pdfmorph
+
+# FIXME - make sure information is correct here
```

Comment:
@chiahaoliu @sbillinge I don't know Soham's GitHub account yet so I can't ping him for more info about this (he said he uses it most out of everyone), but does this description kind of work for how to use the software in a really elementary way? Let me know!

Reply:
soham's account: sbanerjee23

I am not sure about this project but for most of packages I've interacted with, this section is mostly for "do this to verify the software is installed properly" and even the elementary workflow is placed at online documents which allow a more comprehensive view.

Reply:
@sbanerjee23 though we might be putting actual instructions in a separate doc, could you weigh in on whether or not this general description of the process looks ok? goes from lines 69-78

Reply:
yes, this is a bit like the license file.....maintain things in one place and point to that place from elsewhere, though a bit of info for people to get started without reading the docs is good.  Summary: use good judgement.

Reply:
Would it be a good idea to open an issue for creating a manual, like the one [here](https://www.diffpy.org/doc/pdfgetx/2.0.0/PDFgetXN3_manual.pdf)? I think that's broadly the kind of thing we're looking to do with this, right?